lib/apptentive_flutter.dart

```diff
@@ -90,7 +80,7 @@ class ApptentiveFlutter {
     final bool registered = await _channel.invokeMethod('register', {
       "configuration" : _packConfiguration(configuration),
```

Comment:
iOS is expecting `distributionVersion` & `distributionName` being passed separately. The configuration object now have them. I think we can get rid of this.

Reply:
Agreed. We do have to remember to set the version in ApptentiveFlutterPlugin.m since I don't know of a way to get the plugin version at runtime the way we can with e.g. JavaScript environments.

Reply:
Yes. I couldn't get that working in Android as well and passing the hardcoded value. Here I was wondering the passed argument configuration now has the distribution name & version to it @ line number 23 & 24.